### ISSUE TITLE
Injecting into a function with a "ctx" or "callback" argument fails

### ISSUE THREAD

**Issue by lukasthaler** [FasterSpeeding/Alluka#445]
When trying to inject into a function with one of the argument names mentioned in the title, the internal call at https://github.com/FasterSpeeding/Alluka/blob/master/alluka/abc.py#L542 fails, because the function has ctx and callback arguments of its own. This is true for a few more client functions from what I gathered at a glance.
The quick and dirty solution would be to use namespaced versions of internal arguments (i.e. `alluka_context` and `alluka_callback`) to avoid clashing with users trying to use the "ctx" and "callback" variables in their code, but maybe, there is a more elegant solution that I don't see right now

MRE:
```py
import alluka


@alluka.local.auto_inject
def fun(some_arg: int, ctx):
    print(some_arg)
    print(ctx.val)


if __name__ == '__main__':
    client = alluka.Client().set_type_dependency(int, 2)
    alluka.local.initialize(client)
    fun(ctx = 1)
```

**Comment by lukasthaler** [comment]
If the quick fix I mentioned is fine with you, I can get on a PR to make the changes

**Comment by FasterSpeeding** [comment]
Should be fixed by #446
